### PR TITLE
Bolvian wrestlers

### PR DESCRIPTION
**Pull request by jamesgorrie**
# Granular autoplay test

Test to try and determine autoplay use when the video is made for it.
Obviously we know this will be a small subset  - but that's why it seems worth it.
This will make it easy to track other stats such as time on page etc.
## Request for comment

@akash1810 
## 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: common/app/conf/switches/ABTestSwitches.scala [hunk 1]
```diff
@@ -31,6 +31,15 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  val ABBolivianWrestlersAutoplay = Switch(
+    "A/B Tests",
+    "ab-bolivian-wrestlers-autoplay",
+    "Autoplay Bolivian Wrestling",
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 3, 31),
```
Comment (akash1810) [inline common/app/conf/switches/ABTestSwitches.scala]: why so short?

Comment (jamesgorrie) [inline common/app/conf/switches/ABTestSwitches.scala]: 1 week - just to see.

Comment (jamesgorrie) [inline common/app/conf/switches/ABTestSwitches.scala]: There won't be a massive tial on this article.

File: static/src/javascripts/projects/common/modules/experiments/tests/bolivian-wrestling-autoplay.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        this.description = 'Autoplay embedded videos on Bolivian wrestling pages';
+        this.audience = 1;
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = '';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            return window.location.pathname === '/global-development/2016/mar/24/bolivia-cholitas-female-wrestlers-cholitas-discrimination-stranglehold';
```
Comment (akash1810) [inline static/src/javascripts/projects/common/modules/experiments/tests/bolivian-wrestling-autoplay.js]: are we sure we want to autoplay on all platforms?

**Comment by akash1810** [comment]
:+1:

**Pull request merged**